### PR TITLE
[Docs]fix math api en docs issue

### PR DESCRIPTION
**Pull request by ustiniankw**
### PR types

Others
### PR changes

Docs
### Describe

paddle.stanh

### CODE REVIEW

File: python/paddle/tensor/einsum.py [hunk 1]
```diff
@@ -862,160 +862,199 @@ def gen_einsum_op(equation, *operands):
...
+                #     [0.69078457, 0.42282537, 0.30161136]],
+                #
+                #    [[0.32043904, 0.18164253, 0.27810261],
+                #     [0.50226176, 0.24512935, 0.39881429],
+                #     [0.51476848, 0.23367381, 0.39229113]]])
+    <<<<<<< HEAD
+
+    =======
+
+    >>>>>>> 560eed169b9e1b99e4f5041f8dc4b5283862cd7b
```
Comment (SigureMo) [inline python/paddle/tensor/einsum.py]: 这个文件是解决冲突出错了？

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -56,12 +56,12 @@ repos:
...
 # For Python files
 -   repo: https://github.com/psf/black.git
-    rev: 22.8.0
+    rev: 22.10.0
     hooks:
     -   id: black
         files: (.*\.(py|pyi|bzl)|BUILD|.*\.BUILD|WORKSPACE)$
 -   repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
+    rev: 5.0.4
```
Comment (SigureMo) [inline .pre-commit-config.yaml]: 先不要更新版本吧，这些都还原吧

File: python/paddle/distributed/fleet/base/distributed_strategy.py [hunk 3]
```diff
@@ -463,22 +472,30 @@ def trainer_desc_configs(self, configs):
     @property
     def fs_client_param(self):
         """
+
         Set fs client configurations.
+
         **Notes**:
             uri(str): the uri of fs client
```
Comment (SigureMo) [inline python/paddle/distributed/fleet/base/distributed_strategy.py:479]: `**Notes**` -> `Note:`

可参考 https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_docs_guidelines_cn.html#yingwenmuban

File: python/paddle/hapi/model.py [hunk 4]
```diff
@@ -259,13 +259,13 @@ def _update_input_info(inputs):
     return shapes, dtypes
 
 
-class StaticGraphAdapter:
+class StaticGraphAdapter(object):
```
Comment (SigureMo) [inline python/paddle/hapi/model.py]: 解决冲突出问题了？这个是最近刚移除的，这里加回来是有什么理由吗

下面还有很多处

File: python/paddle/hapi/model.py [hunk 5]
```diff
@@ -1756,7 +1772,7 @@ def fit(
                How to make a batch is done internally.
 
             .. code-block:: python
-              :name: code-example1
+              :name: code-example3
```
Comment (SigureMo) [inline python/paddle/hapi/model.py:1777]: 这里为什么要改成 3、4 呢，中文那边还可以正常 copy 过去吗？

File: python/paddle/nn/functional/distance.py [hunk 6]
```diff
@@ -48,10 +49,9 @@ def pairwise_distance(x, y, p=2.0, epsilon=1e-6, keepdim=False, name=None):
 
     Returns:
         Tensor, the dtype is same as input tensor.
-        - If :attr:`keepdim` is True, the output shape is :math:`[N, 1]` or :math:`[1]`,
-            depending on whether the input has data shaped as :math:`[N, D]`.
-        - If :attr:`keepdim` is False, the output shape is :math:`[N]` or :math:`[]`,
-            depending on whether the input has data shaped as :math:`[N, D]`.
+
+        - If :attr:`keepdim` is True, the output shape is :math:`[N, 1]` or :math:`[1]`, depending on whether the input has data shaped as :math:`[N, D]`.
```
Comment (SigureMo) [inline python/paddle/nn/functional/distance.py]: 这里应该在上面加一个空行即可，不需要展开成一行，建议在适当位置折行（目前建议行宽 <= 80）

File: python/paddle/nn/functional/loss.py [hunk 7]
```diff
@@ -1340,7 +1341,7 @@ def l1_loss(input, label, reduction='mean', name=None):
 
     Returns:
         Tensor, the L1 Loss of Tensor ``input`` and ``label``.
-        If `reduction` is ``'none'``, the shape of output loss is [N, *], the same as ``input`` .
+        If `reduction` is ``'none'``, the shape of output loss is [N, `*`], the same as ``input`` .
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]: 建议使用 math

```rst
:math:`[N, *]`
```

File: python/paddle/nn/functional/pooling.py [hunk 8]
```diff
@@ -1894,19 +1900,13 @@ def adaptive_avg_pool3d(x, output_size, data_format='NCDHW', name=None):
...
-        output_size (int|list|tuple): The pool kernel size. If pool kernel size is a tuple or list,
-            it must contain three elements, (D, H, W). D, H and W can be either a int, or None which means
-            the size will be the same as that of the input.
-        data_format (str): The data format of the input and output data. An optional string
-            from: "NCDHW", "NDHWC". The default is "NCDHW". When it is "NCDHW", the data is stored in
-            the order of: [batch_size, input_channels, input_depth, input_height, input_width].
-        name(str, optional): For detailed information, please refer
-                             to :ref:`api_guide_Name`. Usually name is no need to set and
-                             None by default.
+        x (Tensor): The input tensor of adaptive avg pool3d operator, which is a 5-D tensor. The data type can be float32, float64.
```
Comment (SigureMo) [inline python/paddle/nn/functional/pooling.py]: 同上，建议以 4 缩进折行

File: python/paddle/nn/layer/loss.py [hunk 9]
```diff
@@ -26,82 +26,87 @@
...
+                For more information, please refer to :ref:`api_guide_Name`.
 
-    Returns:
-        A callable object of BCEWithLogitsLoss.
+        Shapes:
+    <<<<<<< HEAD
+            - logit (Tensor): The input predications tensor. 2-D tensor with shape: [N, *],
+    =======
+            - logit (Tensor): The input predications tensor. 2-D tensor with shape: attr:`[N, *]`,
+    >>>>>>> 560eed169b9e1b99e4f5041f8dc4b5283862cd7b
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]: 应该是解决冲突出问题了

File: python/paddle/nn/layer/loss.py [hunk 10]
```diff
@@ -26,82 +26,87 @@
...
-    This operator combines the sigmoid layer and the :ref:`api_nn_loss_BCELoss` layer.
-    Also, we can see it as the combine of ``sigmoid_cross_entropy_with_logits``
-    layer and some reduce operations.
 
-    This measures the element-wise probability error in classification tasks
-    in which each class is independent.
-    This can be thought of as predicting labels for a data-point, where labels
-    are not mutually exclusive. For example, a news article can be about
-    politics, technology or sports at the same time or none of these.
+        This operator combines the sigmoid layer and the :ref:`api_nn_loss_BCELoss` layer.
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]: 这里貌似不需要加缩进？

File: python/paddle/tensor/math.py [hunk 11]
```diff
@@ -277,14 +277,13 @@ def stanh(x, scale_a=0.67, scale_b=1.7159, name=None):
 
     .. math::
 
-        out = b * \frac{e^{a * x} - e^{-a * x}}{e^{a * x} + e^{-a * x}}
+        out = b * \\frac{e^{a * x} - e^{-a * x}}{e^{a * x} + e^{-a * x}}
```
Comment (SigureMo) [inline python/paddle/tensor/math.py]: 这个在前面加 r 就可以了，可以参考其他带公式的 docstring

File: python/paddle/vision/ops.py [hunk 12]
```diff
@@ -2478,26 +2483,30 @@ def matrix_nms(
...
-        A tuple with three Tensor: (Out, Index, RoisNum) if return_index is True,
-        otherwise, a tuple with two Tensor (Out, RoisNum) is returned.
-        Out (Tensor): A 2-D Tensor with shape [No, 6] containing the
-             detection results.
-             Each row has 6 values: [label, confidence, xmin, ymin, xmax, ymax]
-        Index (Tensor): A 2-D Tensor with shape [No, 1] containing the
-            selected indices, which are absolute values cross batches.
-        rois_num (Tensor): A 1-D Tensor with shape [N] containing
-            the number of detected boxes in each image.
+        - A tuple with three Tensor: (Out, Index, RoisNum) if return_index is True,
```
Comment (SigureMo) [inline python/paddle/vision/ops.py]: Returns 里带 `:` 应该会显示异常，改成 `,` 吧

File: python/paddle/hapi/model.py (hunk 5 above)
Comment (ustiniankw) [reply python/paddle/hapi/model.py:1777]: 报错信息：Duplicate explicit target name: “code-example1”. 文件中有两个多段代码示例，前面的一个API用了“:name: code-example1”以及“:name: code-example2”

File: python/paddle/nn/functional/pooling.py [hunk 13]
```diff
@@ -1735,15 +1735,17 @@ def adaptive_avg_pool1d(x, output_size, name=None):
 
 def adaptive_avg_pool2d(x, output_size, data_format='NCHW', name=None):
     r"""
+
     Applies 2D adaptive avg pooling on input tensor. The h and w dimensions
     of the output tensor are determined by the parameter output_size.
 
     For avg adaptive pool2d:
+
```
Comment (momozi1996) [inline python/paddle/nn/functional/pooling.py:1743]: 预览中，这公式丢了。avg_pool2d

File: python/paddle/nn/functional/pooling.py [hunk 14]
```diff
@@ -1879,34 +1883,30 @@ def adaptive_avg_pool2d(x, output_size, data_format='NCHW', name=None):
 
 def adaptive_avg_pool3d(x, output_size, data_format='NCDHW', name=None):
     r"""
+
     This operation applies 3D adaptive avg pooling on input tensor. The h and w dimensions
     of the output tensor are determined by the parameter output_size.
 
     For avg adaptive pool3d:
+
```
Comment (momozi1996) [inline python/paddle/nn/functional/pooling.py:1891]: 预览中，这个公式也丢了。avg_pool3d

**Review by momozi1996 — COMMENTED** [review]
除了提到的两个问题，其余我check暂时无问题了。

**Review by momozi1996 — DISMISSED** [review]
LGTM

File: python/paddle/nn/functional/pooling.py (hunk 14 above)
Comment (momozi1996) [reply python/paddle/nn/functional/pooling.py:1891]: 再预览ok

File: python/paddle/nn/functional/pooling.py (hunk 13 above)
Comment (momozi1996) [reply python/paddle/nn/functional/pooling.py:1743]: 再预览ok

File: python/paddle/hapi/model.py (hunk 4 above)
Comment (SigureMo) [reply python/paddle/hapi/model.py]: > 下面还有很多处

这个还没有改，class 继承的 object 可以直接删掉，super() 里面的参数可以直接删掉，后面还有多处，可以在 files changed 里查看具体哪些没有改

**Review by Ligoml — DISMISSED** [review]
LGTM
有一些小问题后续提个pr再改吧

**Review by Ligoml — APPROVED** [review]
good job!
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM
